cmd/opampsupervisor/README.md

```diff
@@ -66,27 +66,27 @@ For a list of open issues related to the Supervisor, see [these issues](https://
-| ReportsPackageStatuses         | 📅                                                                               |
-| ReportsOwnTraces               | 📅                                                                               |
-| ReportsOwnMetrics              | ⚠️                                                                               |
-| ReportsOwnLogs                 | 📅                                                                               |
-| AcceptsOpAMPConnectionSettings | <https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/21043> |
-| AcceptsOtherConnectionSettings | <https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/21043> |
-| AcceptsRestartCommand          | <https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/21077> |
-| ReportsHealth                  | ⚠️                                                                               |
+| ReportsPackageStatuses         | <https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/38727> |
+| ReportsOwnTraces               | ✅                                                                               |
+| ReportsOwnMetrics              | ✅                                                                               |
+| ReportsOwnLogs                 | ✅                                                                               |
+| AcceptsOpAMPConnectionSettings | ✅                                                                               |
+| AcceptsOtherConnectionSettings | ✅                                                                               |
+| AcceptsRestartCommand          | ✅                                                                               |
+| ReportsHealth                  | ⚠️                                                                                |
 | ReportsRemoteConfig            | <https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/21079> |
-| ReportsAvailableComponents     | <https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/37247> |
+| ReportsAvailableComponents     | ✅                                                                               |
 
 ### Supervisor specification features
 
 | Feature                                                            | Status                                                                           |
 |--------------------------------------------------------------------|----------------------------------------------------------------------------------|
 | Offers Supervisor configuration including configuring capabilities | ✅                                                                               |
-| Starts and stops a Collector using remote configuration            | ⚠️                                                                               |
```

Comment:
cc @evan-bradley do you remember if there were any caveats here, and if so, what they were?

I see that if an empty remote config is sent, then we handle the `agentNotStarting` case.

Reply:
Unfortunately, I can't remember what this is about. There are still a few things to be done, but in my opinion, we can consider this to be ready-to-go for now. Thanks for checking.